### ISSUE TITLE
Rubberduck windows not correctly sized

### ISSUE THREAD

**Issue by Irubataru** [rubberduck-vba/Rubberduck#4634]
I am going to guess there is an issue on this already, but I can't find any, so apologies in advanced.

I recently installed rubberduck on a new machine and the rubberduck "interface" seems to be smaller than their containing windows. Here is a picture to illustrate:

![rubberduck_issue](https://user-images.githubusercontent.com/5611071/49929431-c78fca00-fec2-11e8-8c7e-01bc02a879a5.PNG)

I have seen this behaviour on before on my previous install, but it seems to have gone away when I restarted rubberduck. However this time it persists, even after a reboot.

This seems to be an issue with Windows and scaling. My setup at the moment is 1 laptop display and one extra monitor. WIndows is set to 125% scaling, which seems to only affect the laptop display as far as I can see. If I unplug the second display and remain at 125%, this issue goes away (even without restarting rubberduck). If I close rubberduck, set the scaling to 100%, and restart rubberduck, then the issue reappears, even with only the 1 desktop monitor. I haven't tried a full reboot between changing the scaling, so it might just be something residual on Windows' side.

Rubberduck version installed is 2.3.1.4308.

**Comment by Irubataru** [comment]
I tried setting scaling to 100%, then rebooting, and now it works fine, even with a second monitor. I would however prefer to have 125% scaling on my laptop if at all possible.

**Comment by Vogel612** [comment]
Compare #4555 and #4230

**Comment by mansellan** [comment]
Hi @Irubataru - could you try the following please?

If you're on Windows 8.1 or 10:
Go to Rubberduck Settings -> General and select "Set DPI unaware" then restart the application.

If you're on earlier Windows:
Set the DPI scaling in Control Panel to "Use XP-style scaling" (under Advanced options)

**Comment by Irubataru** [comment]
Yeah, that seems to solve the issue. Thanks, and sorry for not noticing the option.

**Comment by bclothier** [comment]
Thanks!  I'm a bit surprised that this exists still even on Excel 2016, we had presumed it was an issue for the older hosts, which is why this fix existed already.  We may need to update the documentation to indicate it may be universally applicable.  Again thanks for the update!

**Comment by mansellan** [comment]
Just a long shot - do have have any other VBE addins loaded alongside RD?

**Comment by Irubataru** [comment]
Yes, I also have the VBA Code Cleaner plugin, but that is all; [this one](http://www.appspro.com/Utilities/CodeCleaner.htm).

**Comment by mansellan** [comment]
Thanks!, We'll try and repro, because it shouldn't be necessary to set that option here - as bclothier noted, it was added for very old VBA hosts.

**Comment by Irubataru** [comment]
It doesn't seem like it was fixed after all. I rebooted my laptop and the problem returned, even with the DPI Unaware setting ticked. I never rebooted between setting the DPI Unaware flag, and it seems as if I need to boot with the interface scaling at 100% for it to work properly. If I then change the interface scaling to e.g. 125% after booting, rubberduck works fine. However, if I reboot with it set to 125%, RD will have these issues no matter what I set the interface scaling to afterwards.

**Comment by bclothier** [comment]
Just to help us rule it out - does it work if you disable the VBA Code Cleaner plugin?

**Comment by Irubataru** [comment]
No, uninstalling it doesn't make a difference.

**Comment by mansellan** [comment]
Windows has a threshold at 125% between two different scaling strategies. Would be interesting to know if this happens at 120%, or 130% (in Custom). Also, you mentioned multiple monitors - is the effect apparent on all of them, or only some?

**Comment by NoKwarl** [comment]
I wanted to add my exact same experience as the poster - I haven't tried the workarounds yet but DPI aware/unaware makes no difference. Latest Office Insider build, Rubberduck V. 2.3.1.4308
